fobi_custom/plugins/form_handlers/collect_data/fobi_form_handlers.py

```diff
@@ -45,11 +45,11 @@ def run(self, form_entry, request, form, form_element_entries=None):
 
             new_survey = Survey.objects.create(**new_survey_info)
 
-            total = self.get_saved_data('total', 1)
+            # total = self.get_saved_data('total', 1)
 
             new_survey_row = SurveyRow.objects.create(
                 survey=new_survey,
-                total=total
```

Comment:
I'd be in favor of removing these lines entirely instead of commenting them out. We can always come back to this PR if we want to reintroduce the functionality.

Reply:
Sorry, this should be attached to the new lines which were commented out instead of the old deleted lines.

Reply:
Makes sense!